.woloxci/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:10.11.0-alpine
```

Comment:
I'd use 12.14.1-alpine, which is the latest stable release: https://nodejs.org/en/